kratos/utilities/math_utils.h

```diff
@@ -1851,7 +1850,7 @@ class MathUtils
         }
 
         // Calculate the solution from the previous decomposition and eigenvalues square root
-        rMatrixSquareRoot = prod(eigenvectors_matrix, Matrix(prod(eigenvalues_matrix, trans(eigenvectors_matrix))));
+        BDBtProductOperation(rMatrixSquareRoot, eigenvalues_matrix, eigenvectors_matrix);
```

Comment:
I search for this one when implementing it and I didn't find it :man_facepalming: